tests/integration_tests/csv_upload_tests.py

```diff
@@ -556,4 +561,4 @@ def test_import_parquet(mock_event_logger):
 
     with test_db.get_sqla_engine_with_context() as engine:
         data = engine.execute(f"SELECT * from {PARQUET_UPLOAD_TABLE}").fetchall()
-        assert data == [("max", 3), ("bob", 4), ("john", 1), ("paul", 2)]
+        assert data == [("john", 1), ("paul", 2), ("max", 3), ("bob", 4)]
```

Comment:
A `SELECT` does not guarantee order without an `ORDER BY`; if we don't want to use `sorted()` here we need to add an explicit `ORDER BY` to the query.

Reply:
@betodealmeida I've re-authored the PR based on your comment.